### PR TITLE
Add MyTeamFunctionality

### PR DESCRIPTION
**Pull request by GloriaXM**
Add functionality to load an individual player's details. Individual users can add players to their MyTeams. MyTeam view displays players currently on the user's team

### CODE REVIEW

File: backend/server.js [hunk 1]
```diff
@@ -73,12 +73,26 @@ app.get("/searchPlayers", async (req, res) => {
...
 
-//TODO: decide if we expect to make a lot of state changes / if it should be stored as state array or in datatbase
 app.get("/myTeamPlayers", async (req, res) => {
-  const players = await prisma.myTeamPlayer.findMany({});
+  const userId = parseInt(req.query.userId);
+  const players = await prisma.myTeamPlayer.findMany({
+    where: {
+      userId,
+    },
+  });
```
Comment (GloriaXM) [inline backend/server.js:82]: Screenshot 2024-07-08 at 9 41 36 AM

File: frontend/src/MyTeam/MyTeamView.jsx [hunk 2]
```diff
@@ -2,14 +2,48 @@ import "./MyTeamView.css";
...
   return (
     <div className="myTeamView">
       <Header />
-      <PlayerCardList myTeamStats={[]} />
-      <StatsTable playersList={[]} />
+      <PlayerCardList
+        myTeamPlayers={myTeamPlayers}
+        playersStats={playersStats}
+      />
+      <StatsTable playersList={playersStats} />
```
Comment (GloriaXM) [inline frontend/src/MyTeam/MyTeamView.jsx:57]: Screenshot 2024-07-08 at 9 57 14 AM

File: frontend/src/MyTeam/PlayerCard.jsx [hunk 3]
```diff
@@ -1,13 +1,30 @@
...
+      headers: {
+        "Content-type": "application/json; charset=UTF-8",
+      },
+    });
+
+    setMyTeamPlayers(
+      myTeamPlayers.filter(function (existingPlayer) {
+        return existingPlayer.id != player.myTeamId;
+      })
+    );
```
Comment (GloriaXM) [inline frontend/src/MyTeam/PlayerCard.jsx:29]: https://github.com/GloriaXM/numbers-game/assets/112780860/bd49154d-981b-406c-a9ea-3cfd90d5023c

File: frontend/src/MyTeam/MyTeamView.jsx (hunk 2 above)
Comment (min2033) [reply frontend/src/MyTeam/MyTeamView.jsx:57]: nit: add paddings to player boxes at the top

File: frontend/src/SinglePlayer/PlayerBanner.jsx [hunk 4]
```diff
@@ -1,14 +1,47 @@
...
+  const playedGames = player[1];
+
   return (
     <div className="playerBanner">
-      <div>
-        <img src="src/assets/movie.png" />
-        <h2> Player Name</h2>
+      <div className="bannerLeft">
+        <h1 className="playerName"> {player[0]}</h1>
+        <h3 className="age">Age: {player[12]}</h3>
```
Comment (min2033) [inline frontend/src/SinglePlayer/PlayerBanner.jsx:10]: "player[12]" these are pretty ambiguous and hard to read / debug. is there a way to structure this in a way you can read it as a field? i.e. player["age"]

Comment (GloriaXM) [reply frontend/src/SinglePlayer/PlayerBanner.jsx:10]: Closing pr to update main branch, but will revisit

**Pull request merged**